documentation/modules/proc-iot-configuring-sigfox.adoc

```diff
@@ -0,0 +1,144 @@
+  are logged in.
+
+[id='iot-proc-configuring-sigfox-registering-backend-gateway-device-{context}']
+== Registering the Sigfox backend as a gateway device
+
+The Sigfox backend will be set up in {ProductName} as a gateway device.
+The credentials assigned to this device will be required for the configuration
+of the "callback" in the Sigfox backend. The actual devices will be configured
+to use this gateway device as their transport.
+
+.Procedure
+
+. link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#iot-proc-creating-device-register-new-device-{context}[Register a new device]. Choose a name, 
+  for example, `sigfox-backend`.
+. link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#iot-proc-creating-device-set-password-{context}[Set up password credentials] for this device (for example `sigfox-user` / `sigfox-password`).
+
+[id='iot-proc-configuring-sigfox-register-sigfox-device-{context}']
+== Registering the Sigfox device
+
+.Prerequisites
+* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#iot-proc-configuring-sigfox-registering-backend-gateway-device-{context}[Registered the Sigfox backend as a gateway device].
+
+.Procedure
+
+. Obtain the _device ID_ for the Sigfox device you want to register. You will get this ID as part of the registration
+  process in the Sigfox backend.
+. link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#iot-proc-creating-device-register-new-device-{context}[Register a new device].
++
+Specify the device ID as the name (for example, `1AB2C3`) and specify name of the gateway device in the `via` field,
+as part of the registration information (for example, `{"via": ["sigfox-backend"]}`).
+
+NOTE: Do not set a password for this device. 
+
+[id='iot-proc-configuring-sigfox-connection-information-{context}']
+== Preparing Sigfox connection information
+
+Prepare the following connection information, which will be used in the
+following steps.
+
+IoT tenant name::
+The name of the IoT tenant consists of the {KubePlatform} namespace and the name
+of the IoT project resource. For example, `namespace.iotproject`.
+
+HTTP authorization header::
+For the Sigfox backend to authenticate
+convert the username and password combination of the gateway device into
+an HTTP "Basic Authorization" header. Be sure to use the full username
+using the following syntax:  `_authentication id_ @_IoT tenant
+name_`.
++
+Example: `sigfox-user@namespace.iotproject`
++
+The basic authentication header value can be generated on the command line using
+the following command:
++
+[options="nowrap",subs="attributes"]
+----
+echo "Basic $(echo -n "sigfox-user@namespace.iotproject:password" | base64)"
+----
+
+URL pattern::
+The URL pattern consists of the URL to the Sigfox protocol adapter and
+Sigfox-specific query parameters:
++
+[options="nowrap",subs="verbatim,attributes"]
+----
+https://<ADAPTER URL>/data/telemetry/<TENANT>?device={device}&data={data}
```

Comment:
Should we provide the an exact example, with `cmd-get-adapter` for example?

Reply:
I am not sure about that. I think this form is easy to understand. And the next paragraphs explain how to obtain the information. I am not sure if an additional code snippet helps.

Reply:
What I had in mind it to have something like this somewhere in the doc:

`export SIGFOX_ADAPTER_HOST=${cmd-get-adapter}`

and then use

`https://$SIGFOX_ADAPTER_HOST`

throughout the docs. We use this for a couple of things already and to me it looks like the most useable/readable approach.